app/src/main/java/me/saket/dank/utils/DankLinkMovementMethod.java

```diff
@@ -24,11 +25,17 @@ public Point getLastUrlClickCoordinates() {
     return clickedUrlCoordinates;
   }
 
+  public Point getLastTouchCoordinates() {
+    return touchCoordinates;
+  }
+
   @Override
   public boolean onTouchEvent(TextView view, Spannable text, MotionEvent event) {
     if (event.getAction() == MotionEvent.ACTION_UP) {
       // A link is potentially going to be clicked.
       clickedUrlCoordinates = new Point((int) event.getRawX(), (int) event.getRawY());
+    } else if (event.getAction() == MotionEvent.ACTION_DOWN) {
+      touchCoordinates = new Point((int) event.getRawX(), (int) event.getRawY());
```

Comment:
Can `clickedUrlCoordinates` not be used here?

Reply:
Hmm good point, didn't think of that. I guess it should work.

Reply:
Updated in 58cb38d51cbef2dafb944a201d6bbad758f217fc